src/controllers/newAccountController.ts

```diff
@@ -0,0 +1,23 @@
+import { User } from '../models/customerModel'
+import { createAccount } from "../models/accountModel";
+import { Account } from "../models/accountModel"
+import { addMoney } from "../models/accountModel";
+import generateAccountNumber from "../utilities/generateAccountNumber";
+
+const buildAccountObject = (user: User): Account => {
+    const accountNumber = generateAccountNumber
+    return {
+        customer_id: user.customer_number,
+        account_type: 'standard',
+        balance: 0,
+        interest_rate: 0,
+        custom_label: 'Main Account',
+        account_number: accountNumber  // Placeholder for function - remember to change!
+    }
+}
+
+export const createFirstAccount = async (user: User) => {
```

Comment:
It would be a good idea to add some error handling here in case the account doesn't get created

Reply:
(Unless you're already handling it in `createAccount`)